app/controllers/public/api/v1/portals_controller.rb

```diff
@@ -0,0 +1,15 @@
+class Public::Api::V1::PortalsController < PublicController
+  before_action :set_portal, except: [:index]
+
+  def index
+    @portals = Portal.all.active
+  end
```

Comment:
How does this work? If it lists all the portals available in the installation, please remove this end point.

We would need the show end point only.